datahub-web-react/src/app/entity/glossaryTerm/profile/GlossaryRelatedTermsResult.tsx

```diff
@@ -12,6 +12,8 @@ import RelatedTerm from './RelatedTerm';
 export enum RelatedTermTypes {
     hasRelatedTerms = 'Contains',
     isRelatedTerms = 'Inherits',
+    containedBy = 'Contained by',
+    isAChildren = 'Inherited by',
```

Comment:
just to be a bit more consistent with `containedBy` what do you think about calling this enum entry `inheritedBy`?

Reply:
actually keys of this enum are implicitly used to address entityData fields here https://github.com/datahub-project/datahub/blob/master/datahub-web-react/src/app/entity/glossaryTerm/profile/GlossaryRelatedTerms.tsx#L61 , and `isAChildren` is a field of type [GenericEntityProperties](https://github.com/datahub-project/datahub/blob/master/datahub-web-react/src/app/entity/shared/types.ts#L98). I can change this enum to structure like this
```
export const RELATED_TERM_TYPES = {
    inheritedBy: {
        title: 'Inherited by',
        fieldName: 'isAChildren',
    },
...
```
How do you think, would that be better? Or maybe just leave it as is?

Reply:
hey great call! I think we can leave it as is in that case.

I wouldn't be opposed to updating `glossaryTerm.graphql` and changing `isAChildren` -> `inheritedBy` and then updating `GenericEntityProperties` and anywhere that references the field `isAChildren`, however I think that's out of scope here and i'm cool with moving forward with what you have.